internal/controllers/machinedeployment/machinedeployment_sync_test.go

```diff
@@ -577,7 +574,7 @@ func TestComputeDesiredMachineSet(t *testing.T) {
 		expectedMS := skeletonMSBasedOnMD.DeepCopy()
 
 		g := NewWithT(t)
-		actualMS, err := (&Reconciler{}).computeDesiredMachineSet(deployment, nil, nil, log)
+		actualMS, err := (&Reconciler{}).computeDesiredMachineSet(ctx, deployment, nil, nil)
```

Comment:
Also this context may not have a logger right?

Reply:
Not sure if it's necessary to ensure a global logger is set so LoggerFrom is falling back to some real logger

Reply:
Fine for me 👍